app/cdap/components/Metadata/SearchResults/ResultList/index.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import styled from 'styled-components';
+import { ISearchResult } from 'components/Metadata/SearchResults/helper';
+import T from 'i18n-react';
+import moment from 'moment';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+
+const I18N_PREFIX = 'features.MetadataSearch';
+
+const Container = styled.ul`
+  background: #fff;
+  list-style: none;
+  padding: 0;
+`;
+
+const List = styled.li`
+  border-bottom: 1px solid #cdcdcd;
+  padding: 8px 15px 15px 15px;
+`;
+
+const Title = styled.h4`
+  font-size: 1.23rem !important;
+  margin: 0;
+  & a {
+    color: #333;
+  }
+`;
+
+const Description = styled.p`
+  color: #999;
+  margin: 0;
+  padding: 0;
+`;
+
+export interface IResultList {
+  results: ISearchResult[];
+  currentPage: number;
+  pageSize: number;
+  query: string;
+}
+
+const ResultList: React.FC<IResultList> = ({ results, currentPage, pageSize, query }) => {
+  const endIndex = currentPage * pageSize;
+  const startIndex = endIndex - pageSize;
+  const pageResults = results.slice(startIndex, endIndex);
+  return (
+    <Container>
+      {pageResults.map((result) => (
+        <List key={result.name}>
+          <Title>
+            <a
+              href={`/metadata/ns/${getCurrentNamespace()}/entity/${result.entityTypeState}/${
+                result.name
+              }/summary?searchTerm=${query}`}
+            >
+              {result.name}
+            </a>
+          </Title>
+          <Description>
+            <span className={result.icon}></span> {result.type}
+          </Description>
+          <Description>
+            {T.translate(`${I18N_PREFIX}.created`)}{' '}
+            {moment(Number(result.createDate)).format('MMM DD, YYYY')}
```

Comment:
Can this use `humanReadableDate`?

Reply:
No, this shows "Sep 30, 2021" but humanReadableDate shows 09-30-2021

Reply:
moment.js is awesome

Reply:
It is - but we should be planning to move off of it. https://momentjs.com/docs/#/-project-status/

We should start centralizing our date formatting logic. See https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/services/DataFormatter/index.ts for an example. What is the input date format here?

Reply:
okay i guess we should start using day.js ? 2kb replacement for moment that is immutable? Is that format service because we have to have generalizable stuff for the abstract widget factory? Thats fine for that but I'd much rather export the individual formatters as well IE `export function formatDateTime`

Reply:
The format service was intended for use in a data table where each cell's type is determined by some metadata. It totally makes sense to split out specific formatters for a use case like this.

We can look at `day.js` but I wonder what the size it is if you bring in internationalized formats. Maybe use `day.js` or similar for parsing and `Intl.DateTimeFormat` for display.

Reply:
Added a helper method based on Intl.DateTimeFormat